third_party/java/arcs/build_defs/internal/util.bzl

```diff
@@ -0,0 +1,10 @@
+"""Shared utilities for Arcs internal rules"""
+
+def output_name(src, suffix = ""):
+    """Cleans up the given file name, and replaces the .arcs extension."""
```

Comment:
I would suggest adding "and replaces the .arcs extension with the provided suffix"